README.md

```diff
@@ -64,9 +64,9 @@ The single required source, file `json.hpp` is in the `single_include/nlohmann`
 using json = nlohmann::json;
 ```
 
-to the files you want to use JSON objects. That's it. Do not forget to set the necessary switches to enable C++11 (e.g., `-std=c++11` for GCC and Clang).
+to the files you want to use JSON objects and set the necessary switches to enable C++11 (e.g., `-std=c++11` for GCC and Clang).
```

Comment:
"JSON objects" sounds odd here (I know it was in my original sentence...). "Object" already has a meaning in JSON - maybe "to the files you want to use JSON in"? Any ideas?

Reply:
Thanks for the feedback. This sounds OK but how about "to the files you want to process JSON". Also, you had used "JSON object" multiple times in this doc.

Reply:
"process JSON" is even better. I have to go through the README again to check where I used "JSON object" in a different sense than "a JSON value of type object"...